deepchecks/nlp/utils/text_properties.py

```diff
@@ -127,10 +193,33 @@ def calculate_default_properties(raw_text: Sequence[str], include_properties: Li
     default_text_properties = _get_default_properties(include_properties=include_properties,
                                                       ignore_properties=ignore_properties)
 
+    # TODO: Avoid running english specific properties if the language is not english
```

Comment:
TODO left here

Reply:
Either solve it in this task or open a new issue but let's not add todos to the code :)

Reply:
https://github.com/deepchecks/deepchecks/issues/2414